lib/storage_tables/services/configurator.rb

```diff
@@ -4,14 +4,14 @@
 require "active_storage/service/configurator"
 
 module StorageTables
-  module Service
+  module Services
```

Comment:
We kunnen het ook een class houden. ActiveStorage heeft het ook zo georganiseerd met nested classes. Ben daar normaal niet zo'n voorstander van, maar ik denk dat het overzichtelijker blijft als we dicht bij ActiveStorage blijven

```suggestion
  class Service
```

Reply:
Check heb dat nu omgezet.